.gitignore

```
@@ -59,6 +59,8 @@ llama-batched-swift
 /rpc-server
 out/
 tmp/
+ggml-vulkan-shaders.hpp
+ggml-vulkan-shaders.cpp
```

These should be ignored in `ggml/.gitignore`

```suggestion
src/ggml-vulkan-shaders.hpp
src/ggml-vulkan-shaders.cpp
```